### DISCUSSION TITLE
Endlessly checks for updates

### DISCUSSION THREAD

**Discussion by vanyuha-pro** [LegacyUpdate/LegacyUpdate#85] [Questions & Help]
Faced with such a problem that Legacy update has been checking for updates for more than 7 hours. On December 6, he did the same thing in just 30 minutes. After installing some updates (December 6), I check them again and the green bar just runs endlessly. Windows xp sp3.

**Comment by Skywatcher17** [comment]
I have this same problem. Versions 1.5 and 1.4.1 crash or don't open at all. Version 1.4 gets to the checking updates stage and then gets no further.

**Reply by ghost** [reply]
The same problem persists in 1.6.2

**Reply by kirb** [reply]
Do you get ":warning: Error on page" in the Internet Explorer status bar by any chance? If so, would appreciate if you could double-click that and take a screenshot (or transcribe the error info it shows in the rectangle area). Sometimes, refreshing might fix it.

**Reply by vanyuha-pro** [reply]
I don't have any mistake. Legacy update hangs on the update search page and nothing happens

**Reply by ghost** [reply]
No error on the address bar, that I can see. (I am enclosing a screen shot.) Something has changed since I last tried to run the updater. The home & Update history buttons are no longer unresponsive (grayed out), however, the update history does the same endless search and never finds any of the updates in history. I don't know what the "home" button is supposed to do but in my case it just goes into the same endless search. (Also, restore hidden updates is not available)
![LUSC](https://user-images.githubusercontent.com/123204066/216776761-2d951b7f-6ef4-4a09-92e1-df0ada87e16a.JPG)

**Reply by KiasAttila** [reply]
I had the same issue, but it started to occur for me after i installed office 2010.
Is there any fix for this?

**Reply by kirb** [reply]
That’s very interesting, I wonder if Office 2010 causes the problem to come back…

**Reply by KiasAttila** [reply]
But after i installed it ot searched for updates found and installed them.
Then it restarted and it is on the infinite searching for updates.

**Reply by kirb** [reply]
Different combinations of non-installed updates can cause it, and (annoyingly) often specific combinations of updates can fix it. I can definitely see that being the case. I’ll see what can be done, possibly manually installing a cumulative update for Office 2010 from about 2016 or later (which you’d need to search for and download manually from https://catalog.update.microsoft.com/) can fix it.

**Reply by rouyng** [reply]
I am also experiencing this issue with a fresh install of Windows Server 2003 Standard Edition, Version 5.2.3790 Service Pack 2 Build 3790. Checking for updates hangs forever. I left it running overnight and noticed that it did also try to automatically install updates, as my "Update History" now has 155 failed updates, all showing "Installation Failure" with "Error Code:80080008 (-2146959352)". BTW, Office is not installed.

[Here's my windows update log](https://gist.github.com/rouyng/cfd438a8603c6ce5c87b6b37a5e8573f)
